modules/service/src/main/scala/ch/epfl/bluebrain/nexus/admin/service/encoders/organization.scala

```diff
@@ -20,19 +22,19 @@ object organization {
     Encoder.encodeJson.contramap { resource =>
       val ld =
```

Comment:
isn't that a weird formatting from scalafmt? `val ld = resource.value.appendContext(resourceContext)` should fit in one line, right?